uphold-integration/src/main/res/values/strings-uphold.xml

```diff
@@ -6,20 +6,20 @@
     <string name="uphold_buy_dash_with_uphold">Buy Dash with Uphold account</string>
     <string name="uphold_buy_dash_button">Buy Dash</string>
     <string name="uphold_transfer_to_this_wallet_button">Transfer from Uphold</string>
-    <string name="uphold_link_account">Link Uphold account</string>
+    <string name="uphold_link_account">Link Uphold Account</string>
     <string name="uphold_account_dash_balance">Your Uphold account Dash balance is</string>
     <string name="uphold_transfer_to_this_wallet">Transfer Dash from your Uphold account to this wallet</string>
     <string name="uphold_withdrawal_deduct_fee_disclaimer">Fee will be deducted from requested amount.</string>
-    <string name="uphold_withdrawal_instructions">Enter the amount to transfer.</string>
+    <string name="uphold_withdrawal_instructions">Enter the amount to transfer</string>
     <string name="uphold_withdrawal_success_title">Success</string>
     <string name="uphold_withdrawal_success_message">Your transaction was sent and the amount should appear in your wallet in a few minutes.\n\nTransaction id: %s</string>
-    <string name="uphold_otp_dialog_title">Enter your 2FA code below:</string>
+    <string name="uphold_otp_dialog_title">Enter your 2FA code below</string>
     <string name="uphold_transfer">Transfer</string>
     <string name="uphold_see_on_uphold">See on Uphold</string>
-    <string name="uphold_logout_title">Log Out from Uphold</string>
+    <string name="uphold_logout_title">Log out from Uphold</string>
```

Comment:
On buttons, we are doing sentence case, which is capitalize the first word and any proper nouns and names.